internal/proto/local/controller/custom_options/v1/options.proto

```diff
@@ -15,6 +15,9 @@ extend google.protobuf.FieldOptions {
 	// generate_sdk_option is a directive used when generating the SDK to
 	// indicate that an option should be created for the field
 	bool generate_sdk_option = 85412;
+
+	// attrs_for_type is used for oneOf attribute subtypes
+	string attrs_for_type = 85555;
```

Comment:
I am thinking we might just want to make this `subtype` since when it is used it will look like:

```proto
[(custom_options.v1.subtype) = "<subtype string>"]
```

Reply:
Todo: Add more detail to this option comment as work fleshes out